datafusion/Cargo.toml

```diff
@@ -66,6 +67,8 @@ regex = { version = "^1.4.3", optional = true }
 lazy_static = { version = "^1.4.0", optional = true }
 smallvec = { version = "1.6", features = ["union"] }
 rand = "0.8"
+pyo3 = { version = "0.14", optional = true }
+libc = { version = "0.2", optional = true }
```

Comment:
Are you now using something from `libc`? I didn't catch that in the first pass.

Reply:
Used for the C data interface between rust and pyarrow.